src/components/AccordionMenu/index.spec.tsx

```diff
@@ -70,6 +71,7 @@ describe(__filename, () => {
         children: <div className={childClass} />,
       });
 
+      expect(root.find(Button)).toHaveLength(1);
```

Comment:
I'm not sure we actually need this. I don't think it buys us anything and it doesn't relate directly to the test in question.